src/server_manager/messages/en.json

```diff
@@ -8,6 +8,8 @@
   "aws-lightsail-firewall-4": "Set 'Application' value to 'All TCP+UDP'.",
   "aws-lightsail-firewall-5": "Click 'Save'.",
   "cancel": "Cancel",
+  "outline-documentation": "Outline Documentation",
+  "server-management-api-documentation": "Server Management API Documentation",
```

Comment:
I think "Server Management" is redundant here, given the field title. It could be "API Documentation".